README.md

```diff
@@ -153,6 +153,29 @@ JSX code blocks in the markdown file can be made interactive with the live attri
 
 Visit the documentation at [http://localhost:8000](http://localhost:8000) and navigate to see your README.md powered page and workbench. Changes to the README.md file will auto refesh the page.
 
+### Developing locally against MFE
+
+If you want to test the changes with local MFE setup, you need to create a "module.config.js" file in your MFE's directory containing local module overrides. After that the webpack build for your application will automatically pick your local version of Paragon and use it. The example of module.config.js file looks like this (for more details about module.config.js, refer to the [frontend-build documentation](https://github.com/edx/frontend-build#local-module-configuration-for-webpack).):
+
+```javascript
+module.exports = {
+  /*
+  Modules you want to use from local source code. Adding a module here means that when 
+  your MFE runs its build, it'll resolve the source from peer directories of the app.
+
+  moduleName: the name you use to import code from the module.
+  dir: The relative path to the module's source code.
+  dist: The sub-directory of the source code where it puts its build artifact. Often "dist".
+  */
+  localModules: [
+    { moduleName: '@edx/paragon/scss/core', dir: '../src/paragon', dist: 'scss/core' },
+    { moduleName: '@edx/paragon/icons', dir: '../src/paragon', dist: 'icons' },
+    { moduleName: '@edx/paragon', dir: '../src/paragon', dist: 'dist' },
```

Comment:
If may also be worth calling out that using `dist: 'dist'` will require you to `npm build` in Paragon before any local changes are built/added to the `dist` directory. I believe you could also be able to use `dist: 'src'` and get any local changes hot reloaded on save in the MFE.

Might be worth calling out that both `dist` and `src` are valid options here?

Reply:
Yeah, I think that's definitely a good idea, haven't thought about it previously, thanks!

updated in 4d8a6a3